pollux/sql-doobie/src/main/resources/sql/pollux/V24__add_resolution_method_to_schema_and_cred_definition.sql

```diff
@@ -0,0 +1,10 @@
+-- Create the enum type
+CREATE TYPE resolution_method_enum AS ENUM ('http', 'did');
```

Comment:
We need to rename `did` to `did:prism` as this resolution method is our domestic specification. What do you think, @shotexa ?